docs/decisions/0013-backend-for-frontend-api.md

```diff
@@ -0,0 +1,69 @@
+# 0013. Introduction of the backend-for-frontend orchaestration API
+
+## Status
+
+Accepted (December 2024)
+
+## Context
+
+As part of supporting the feature of auto enrollment of default enrollment intentions ([see ADR](https://github.com/openedx/edx-enterprise/blob/master/docs/decisions/0015-default-enrollments.rst))
+and auto-apply of subscription plan licenses, it was determined the sequencing of API calls to resolve in the existing 
+frontend root loader would result in a new set of waterfall API calls on top of existing reconciliation of the current status of
+the user's subsidies when arriving on the enterprise learner portal. Furthermore, in support of including this feature,
+it required additional business logic to be parsed about the current state of the requester's association to the enterprise,
+the current state of their subsidies (and auto-apply of subscription licenses), and the eventual auto enrollment of a 
+default enrollment intention.
+
```

Comment:
It might also be worth expanding on the long-term performance rationale for wanting to abstract the API orchestration from the frontend in favor of a primary single API endpoint per page route, starting with the Dashboard.